src/globemap.ts

```diff
@@ -1297,5 +1247,112 @@ module powerbi.extensibility.visual {
             MercartorSphere.prototype = Object.create(THREE.Geometry.prototype);
             GlobeMap.MercartorSphere = MercartorSphere;
         }
+
+        private updateBarsAndHeatMapByZoom(delta: number = 0): void {
+            if (!this.data) {
+                return;
+            }
+            // delta > 0 ? Zoom increased
+            // delta < 0 ? Zoom decreased
+            let heatSizeScale: number = delta > 0 ? GlobeMap.GlobeSettings.heatmapScaleOnZoom : (1 / GlobeMap.GlobeSettings.heatmapScaleOnZoom);
```

Comment:
What is GlobeSettings? I don't think that it's good to change value of static variable.